misc/FFNx.toml

```diff
@@ -550,6 +550,12 @@ save_path = "save"
 # OPTIONS ONLY FOR FF8
 ###############################################################################
 
+## ENHANCEMENTS
+
+# Load higher resolution textures on worldmap
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ff8_worldmap_texture_fix = true

```

Comment:
Would you mind changing the name of this flag to something like `ff8_worldmap_internal_highres_textures` to reflect the real purpose?